cognite/neat/_rules/_constants.py

```diff
@@ -42,7 +43,7 @@ class EntityTypes(StrEnum):
     space = "space"
 
 
-def get_reserved_words() -> dict[str, list[str]]:
+def get_reserved_words(key: Literal["class", "view", "property", "space"]) -> list[str]:
```

Comment:
This was a change I did in preparation for having a mode on the prefix method to only prefix reserved keywords. Decided that it an unnecessary complexity, but the change is a simplification so think we should keep it.